site/tests/universal/homepage.test.ts

```diff
@@ -26,4 +26,40 @@ test("Home page should contain key elements", async ({ page }) => {
       "text=The Block Protocol makes building composable interfaces easy",
     ),
   ).toBeVisible();
+
+  await expect(page.locator("text=Explore all Blocks")).toHaveAttribute(
+    "href",
+    "/hub",
+  );
+
+  await expect(page.locator("text=start building it today")).toHaveAttribute(
+    "href",
+    "/docs/developing-blocks",
+  );
+
+  // Block slider
+  await expect(page.locator('[data-testid="block-slider"]')).toBeVisible();
+
+  // New blocks can get added, hence the usage of greater than instead of equal too
+  expect(
+    await page.locator('[data-testid="block-slider"] >> .slick-slide').count(),
+  ).toBeGreaterThan(5);
+
+  const finalCTA = page.locator('[data-testid="final-cta"]');
```

Comment:
Optional, minor (can be ignored or addressed in a follow-up): `finalCTA` → `finalCta`, i.e. `strictCamelCase`. This naming style is a bit more readable, especially when there are multiple abbreviations next to each other.

Reply:
Will address in a follow up